apps/condo/pages/property/[id]/index.tsx

```diff
@@ -29,7 +29,7 @@ import { Property } from '@condo/domains/property/utils/clientSchema'
 
 const PROPERTY_PAGE_CONTENT_ROW_GUTTER: RowProps['gutter'] = [12, 40]
 const PROPERTY_PAGE_CONTENT_ROW_INFO_BLOCK_GUTTER: RowProps['gutter'] = [52, 24]
-const PROPERTY_PAGE_CONTENT_ROW_INFO_BLOCK_STYLE: React.CSSProperties = { marginTop: '80px' }
+const PROPERTY_PAGE_CONTENT_ROW_INFO_BLOCK_STYLE: React.CSSProperties = { marginTop: '80px', marginRight: '-20px' }
```

Comment:
Why it so? There are no special conditions for display. Does it look ok with negative margin?

Reply:
Yes, initially there margin `-26px` and appears horizontal scroll